modules/sample/src/test/ts/client/AsyncFailTest.ts

```diff
@@ -1,10 +1,26 @@
 import { UnitTest } from '@ephox/bedrock-client'
 
-UnitTest.asynctest('AsyncFail Test', (success, failure) => {
+UnitTest.asyncTest('AsyncFail Test 1', (success, failure) => {
```

Comment:
This test still needs to import a promise polyfill from somewhere.

- [ ] Add Promise polyfill in this test